src/Invariants/Equality.jl

```diff
@@ -0,0 +1,43 @@
+using Graphs
+
+"""
+equals method for undirected edges::Union{Vector{Tuple{Int, Int}}, Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}}
+"""
+function equals(
+    a::Union{Vector{Tuple{Int, Int}}, Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}},
+    b::Union{Vector{Tuple{Int, Int}}, Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}}
+)
+
+    # convert a to an array of tuples
+    if typeof(a) == Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}
+        a = [(min(e.src, e.dst), max(e.src, e.dst)) for e in a]
+    else
+        a = [(min(e[1], e[2]), max(e[1], e[2])) for e in a]
+    end
+
+    # convert b to an array of tuples
+    if typeof(b) == Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}
```

Comment:
Per my comment at the top (the one defining `SimpleInt64Edge`):

```julia 
isa(b, SimpleInt64Edge)  && b = [(min(e.src, e.dst), max(e.src, e.dst)) for e in b]
!isa(b, SimpleInt64Edge) && b = [(min(e[1], e[2]), max(e[1], e[2])) for e in b]
```

Reply:
`isa` is a really useful julia function for checking types. I actually prefer 

```julia 
b isa SimpleInt64Edge
```

over 

```julia 
isa(b, SimpleInt64Edge)
```

But because we had a negation, I decided on the latter